### DISCUSSION TITLE
Why can't I download it

### DISCUSSION THREAD

**Discussion by TeamMeng** [LazyVim/LazyVim#2076] [Q&A]
处理 /Users/mengzhilin/.config/nvim/init.lua 时发生错误:
E5113: Error while calling lua chunk: /Users/mengzhilin/.config/nvim/lua/config/lazy.lua:9: module 'lazy' not found:
        no field package.preload['lazy']
        no file './lazy.lua'
        no file '/opt/homebrew/share/luajit-2.1/lazy.lua'
        no file '/usr/local/share/lua/5.1/lazy.lua'
        no file '/usr/local/share/lua/5.1/lazy/init.lua'
        no file '/opt/homebrew/share/lua/5.1/lazy.lua'
        no file '/opt/homebrew/share/lua/5.1/lazy/init.lua'
        no file './lazy.so'
        no file '/usr/local/lib/lua/5.1/lazy.so'
        no file '/opt/homebrew/lib/lua/5.1/lazy.so'
        no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
        [C]: in function 'require'
        /Users/mengzhilin/.config/nvim/lua/config/lazy.lua:9: in main chunk
        [C]: in function 'require'
        /Users/mengzhilin/.config/nvim/init.lua:2: in main chunk

**Comment by dpetka2001** [comment]
It seems that your lazy installation must not have completed correctly. Could you try deleting directory `~/.local/share/nvim` and try again?

**Reply by TeamMeng** [reply]
get it

**Comment by baoyuxu** [comment]
Hi, did you solve this error?

**Reply by TeamMeng** [reply]
Yes
